README.adoc

```diff
@@ -1,19 +1,58 @@
 = Cloud Enablement module
 
-This repository serves as a temporary public link:https://github.com/concrt/concreate[Concreate] module for building JBoss Openshift container images. This repository is an enhanced copy of shell scripts used to build JBoss images for OpenShift.
+This is a repository of link:http://cekit.io[Cekit] modules, shared by a number
+of related link:https://openshift.org[OpenShift] container images.
+
+== Update process
+
+Releases of this repository are coordinated over a four-week sprint cycle,
+divided into three weeks of development and one week of testing/release
+preparation. Sprint cycles overlap by one week: the first week of development
+coincides with the testing/release phase of the previous sprint. A reference
```

Comment:
Oh, I think this is the part that we never really did, but thats a good point, lets leave it like this.